tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+    assert delete_cephfilesystem(file_name=TEMP_YAML_FILE_FS)
+    assert delete_secret(secret_list)
+
+
+def delete_pvc(pvc_list):
+    """
+    Function to delete pvc
+    :param pvc_list:
+    :return:
+    """
+    for item in pvc_list:
+        log.info(f"Deleting pvc {item}")
+        assert PVC.delete(resource_name=item)
+    return True
+
+
+def delete_storageclass_and_cephblockpool(storageclass_list):
+    """
+    Function for deleting Storageclass and CephBlockPool
+    :param storageclass_list:
+    :return:
+    """
+    for item in storageclass_list:
+        log.info(f"Deleting StorageClass with name {item}")
+        assert SC.delete(resource_name=item)
+        if "cephfs" not in item:
+            pool_name = item.strip('ocsci-csi-')
+            assert POOL.delete(resource_name=pool_name)
+    return True
+
+
+def delete_cephfilesystem(file_name):
+    """
+    Function to delete CephFileSystem
+    :param file_name:
+    :return:
+    """
+    log.info("Deletimg CephFileSystem")
+    assert CFS.delete(yaml_file=file_name)
+    return True
+
+
+def delete_secret(secret_list):
+    """
+    Function to delete Secret
+    :param secret_list:
+    :return:
+    """
+    for item in secret_list:
+        log.info(f"Deleting secret {item}")
+        assert SECRET.delete(resource_name=item)
+    return True
+
+
+def create_rbd_pool(pool_name):
+    """
+    Create Blockpool with specified name
+
+    :param pool_name:
+    :return:
+    """
+    pool_data = {}
+    pool_data['rbd_pool'] = pool_name
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        RBD_POOL_YAML, **pool_data
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new CephBlockPool with name {pool_name}")
+    assert POOL.create(yaml_file=TEMP_YAML_FILE)
+    return True
+
+
+def validate_pool_creation(pool_name):
+    """
+    Check wether Blockpool and pool is created or not at ceph and as well
+    k8s side
+
+    :param pool_name:
+    :return:
+    """
+    ceph_validate = False
+    k8s_validate = False
+    cmd = "ceph osd lspools"
+    out = ocp.exec_ceph_cmd(cmd)
+    for item in out:
+        if item['poolname'] == pool_name:
+            ceph_validate = True
+    pool_obj = POOL.get()
+    sample = pool_obj['items']
+    for item in sample:
+        if item.metadata.name == pool_name:
+            k8s_validate = True
+    if ceph_validate and k8s_validate:
+        return True
+    else:
+        if not ceph_validate:
+            log.error(f"{pool_name} pool wasn't created at CEPH side")
+        if not k8s_validate:
+            log.error(f"{pool_name} pool wasn't created at K8S side")
+        return False
+
+
+def validate_storageclass(sc_name):
+    """
+    Validate if storageClass is been created or not
+    """
+    sc_obj = SC.get()
+    sample = sc_obj['items']
+    for item in sample:
```

Comment:
why not use get(resource_name=sc_name) and that's it?

Reply:
done